glib2/test/test_file_utils.rb

```diff
@@ -3,14 +3,26 @@
 class TestGLibFileUtils < Test::Unit::TestCase
   include GLibTestUtils
 
-  def test_format_size_for_display
-    only_glib_version(2, 16, 0)
-
-    assert_equal("1.0 KB", GLib.format_size_for_display(1024))
-    assert_equal("10.0 KB", GLib.format_size_for_display(1024 * 10))
```

Comment:
Why is this case removed?
Is the case needed for confirming unit (1000 or 1024)?